common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4271,7 +4271,8 @@
   },
   "STREAM_FAILED" : {
     "message" : [
-      "Query [id = <id>, runId = <runId>] terminated with exception: <message>"
+      "Query [id = <id>, runId = <runId>, startOffset = <startOffset>, endOffset = <endOffset>] terminated with exception: <message>",
+      "<queryDebugString>"
```

Comment:
@HeartSaVioR Could you look at the changes, please.

Reply:
This seems to be slightly messed up -

common/utils/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryException.scala

This class overrides getMessage and handles queryDebugString separately. So it's expected to not have queryDebugString here.

What I'm confused is why we have a case where queryDebugString is passed over message parameters, since we have constructors which directly receive queryDebugString.

Reply:
> So it's expected to not have queryDebugString here.

@HeartSaVioR ok, let me remove it from message parameters and from the error format. How about other 2 unused parameters: `startOffset` and `endOffset`. Does `queryDebugString` include the info, it seems?